cmd/internal/ec2/add.go

```diff
@@ -63,6 +72,54 @@ func setupAgentForServer(collectionId string) error {
 	return nil
 }
 
+func askToReconfigure() error {
+	printer.Infof("postman-lc-agent is already present as a systemd service\n")
+	printer.Infof("Helpful commands \n Check status: systemctl status postman-lc-agent \n Disable agent: systemctl disable --now postman-lc-agent \n Check Logs: journalctl -fu postman-lc-agent\n Check env file: cat %s \n Check systemd service file: cat %s \n", envFilePath, serviceFilePath)
+	err := survey.AskOne(
+		&survey.Input{
+			Message: "Over-write old apiKey and collectionId with current values ? (y/n)",
+			Help:    "Enter y(default) or n.",
+			// Use the existing value as the default in case we repeat this step
+			Default: "y",
+		},
+		&isReconfigure,
+	)
+	if err != nil {
+		return errors.Wrap(err, "failed to run reconfiguration prompt")
+	}
+	if isReconfigure != "y" {
+		return errors.New("Exiting process because you didn't enter y")
+	}
```

Comment:
I think this is not an error, and it shouldn't be shown to the user as if it is.